lang/ru.xml

```diff
@@ -19,114 +19,111 @@
 		<developers>Разработчики</developers>
-		<contributors>Contributors</contributors>
+		<contributors>Участники</contributors>
 	</about>
 
 	<app_context>
-		<boot>Boot</boot>
-		<check_app_compatibility>Check App Compatibility</check_app_compatibility>
-		<copy_vita3k_summary>Copy Vita3K Summary</copy_vita3k_summary>
-		<open_issue>Open Issue</open_issue>
-		<create_issue>Create Issue</create_issue>
-		<update_database>Update Database</update_database>
-		<copy_app_info>Copy App Info</copy_app_info>
-		<name_and_id>Name and Title ID</name_and_id>
-		<app_summary>App Summary</app_summary>
-		<custom_config>Custom Config</custom_config>
-		<create>Create</create>
-		<edit>Edit</edit>
-		<remove>Remove</remove>
-		<open_folder>Open Folder</open_folder>
-		<addcont>DLCs</addcont>
-		<license>License</license>
-		<shader_cache>Shader Cache</shader_cache>
-		<shader_log>Shader Log</shader_log>
-		<update_history>Хронология обновлений</update_history>
+		<boot>Запустить</boot>
+		<check_app_compatibility>Проверить совместимость приложений</check_app_compatibility>
+		<copy_vita3k_summary>Скопировать лицензию Vita3K</copy_vita3k_summary>
+		<open_issue>Открыть вопрос</open_issue>
+		<create_issue>Создать вопрос</create_issue>
+		<update_database>Обновление базы данных</update_database>
+		<copy_app_info>Скопировать инвормацию приложения</copy_app_info>
+		<name_and_id>Имя и идентификатор</name_and_id>
+		<app_summary>Лицензия приложения</app_summary>
+		<custom_config>Пользовательская конвигурация</custom_config>
+		<create>Создать</create>
+		<edit>Редактировать</edit>
+		<remove>Удалить</remove>
+		<open_folder>Открыть файлы</open_folder>
+		<addcont>Дополнения</addcont>
+		<license>Лицензия</license>
+		<shader_cache>Кэш шейдеров</shader_cache>
+		<shader_log>Журнал шейдеров</shader_log>
+		<update_history>История обновлений</update_history>
 		<history_version>Версия {}</history_version>
 		<information>Информация</information>
-		<app_delete>Это приложение и все связанные с ним данные, в том числе сохраненные данные, будут удалены.</app_delete>
-		<app_delete_note>Deleting a application may take a while
-depending on its size and your hardware.</app_delete_note>
-		<save_delete>Удалить эти сохраненные данные?</save_delete>
+		<app_delete>Это приложение и все связанные с ним данные, включая сохраненные данные, будут удалены.</app_delete>
+		<app_delete_note>
+			Удаление приложения может занять некоторое время
+			в зависимости от его размера и вашего оборудования.
+		</app_delete_note>
+		<save_delete>Вы хотите удалить эти файлы сохранения?</save_delete>
 		<eligible>Допущен</eligible>
 		<ineligible>Не допущен</ineligible>
 		<level>Уровень</level>
-		<name>Название</name>
-		<trophy_earning>Полученные призы</trophy_earning>
+		<name>Имя</name>
+		<trophy_earning>Доступные трофеи</trophy_earning>
 		<parental_controls>Родительский контроль</parental_controls>
-		<updated>Обновлен</updated>
+		<updated>Обновлено</updated>
```

Comment:
I need check it, @Croden1999 if you can look it please

Reply:
> I need check it, @Croden1999 if you can look it please

i remember is you add it, Mr.Zangetsu, no any problem, he need revert.

Reply:
Yes, but i said check for see for i have no fail in copy past lol

Reply:
> Yes, but i said check for see for i have no fail in copy past lol

I check again, the last "o" no exist